tests/unit/core/models/test_extents.py

```diff
@@ -340,5 +341,18 @@ def test_type_checking(self):
         """
         input = "bad input"
 
-        with self.assertRaisesRegex(TypeError, "Expect polygon, received <class 'str'>"):
-            SpatialExtent(input)
\ No newline at end of file
+        with self.assertRaisesRegex(TypeError, "Expected polygon, received <class 'str'>"):
+            SpatialExtent(input)
+
+    def test_type_checking_crs(self):
```

Comment:
very minor thing, but as SpatialExtent has multiple methods, it can be useful to identify what method is being tested in the test name, e.g. because it's the `__init__` method being tested:
```suggestion
    def test_init_type_checking_crs(self):
```

As the test class grows larger, this becomes more useful for identifying which portion of the class is being tested